### ISSUE TITLE
eXoOS2

### ISSUE THREAD

**Issue by Torinde** [exoscoriae/eXoWin3x#682]
From the [~400 native OS/2 games](https://www.os2world.com/games/index.php/native-games) it seems there are at least a few exclusive ones or such that have features exclusive to their version for OS/2.

On the other hand - part of the games may run under WinNT/2K (#680) via [Microsoft OS/2 subsystem](https://en.wikipedia.org/wiki/Architecture_of_Windows_NT#OS/2_environment_subsystem) and/or [Microsoft OS/2 Presentation Manager](https://winworldpc.com/product/microsoft-os2-presen/for-nt-40).

Emulation-wise OS/2 is from the Win3.x/9x period and should work in PCem/86box/PCbox.

**Comment by martiniturbide** [comment]
FYI
I have this freeware/open/shareware collection of OS/2 games.
https://archive.org/details/OHFOWG

We hang out at os2world.com

**Comment by Torinde** [comment]
@martiniturbide, great!
Do you have a list of games that work ONLY in OS/2 and NOT in Windows? I would assume those will be a priority.

**Comment by martiniturbide** [comment]
Hello @Torinde 

The list that you see on "https://www.os2world.com/games/index.php/native-games" are native compiled games on OS/2. They can be original with the OS/2 API (Presentation manager) or ported games with newer libraries like Qt or SDL. 

I guess you want the list of the original OS/2 games, and not the ones that had been ported from other platforms, right?
(not SDL ported games and not Qt ported games)

Regards

**Comment by Torinde** [comment]
Yes, it would be great if you can mark somehow the games that are not ports from other platforms.

As next step, to mark those that are not available outside OS/2 at all, e.g. the reverse (porting from OS/2 into Windows) was never done.

Further, if some OS/2 (original or port) game has improvements/differences from the other platforms where it runs - that's relevant too (for example the OS/2 port has extra levels or something).

**Comment by francoisjacques** [comment]
The non sharewares are likely to be on archive.org as well (Galactic Civilization, for example)
